service_storagefacility/app/models/event/move/MoveEvents.scala

```diff
@@ -73,13 +89,17 @@ case class MoveNode(
     eventType: EventType,
     from: Option[StorageNodeDatabaseId],
     to: StorageNodeDatabaseId
-) extends MoveEvent
+) extends MoveEvent {
+
+  override val objectType: ObjectType = Node
```

Comment:
Isn't this a code smell? That we need to relate to objectType when moving a node? Or is this intended? If so, should it be documented?